src/Utilities/mockoptimizer.jl

```diff
@@ -275,28 +284,33 @@ function mock_primal!(optimizer::MockOptimizer, varprim::Vector, dual...)
 end
 function mock_primal!(optimizer::MockOptimizer, conduals::Pair...)
     # No primal solution
+    optimizer.hasprimal = false
     MOI.set!(optimizer, MOI.PrimalStatus(), MOI.InfeasiblePoint)
     mock_dual!(optimizer, MOI.InfeasibilityCertificate, conduals...)
 end
 
 # Adds default dual status if its is missing
 dual_args(dualstatus::MOI.ResultStatusCode, conduals::Pair...) = dualstatus, conduals...
+dual_args() = tuple()
```

Comment:
What does this do? How is it used?

Reply:
You are right this isn't clear. I have got rid of `dual_args` in 1851471